linux-anvil-ppc64le/Dockerfile

```diff
@@ -30,6 +30,28 @@ RUN yum update -y && \
 COPY scripts/run_commands /opt/docker/bin/run_commands
 RUN /opt/docker/bin/run_commands
 
+# Download and cache new compiler packages.
+# Should speedup installation of them on CIs.
+RUN source /opt/conda/etc/profile.d/conda.sh && \
+    conda activate && \
+    conda create -n test --yes --quiet --download-only \
+        defaults::binutils_impl_linux-ppc64le \
+        conda-forge::binutils_linux-ppc64le \
```

Comment:
Is there a reason some of these come from `defaults` and some from `conda-forge`? Noticing this differs compared to `comp7` and `cuda` cases above.

Reply:
Yes, we don't build them yet. See https://gitter.im/conda-forge/conda-forge.github.io?at=5db0bf13e469ef435851749f

Reply:
Ok, but we are building everything else?

Reply:
Yes